tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/ServiceRecipients GetAll.feature

```diff
@@ -0,0 +1,54 @@
+﻿Feature: Display all of the Service Recipients for an order
+    As a Buyer
+    I want to view all of the service recipients for an order
+    So that I can ensure the information is correct
+
+Background:
+    Given Orders exist
+        | OrderId    | Description         | OrderStatusId | LastUpdatedByName | LastUpdatedBy                        | OrganisationId                       |

```

Comment:
To reduce the amount of information & to hopefully make the code easier to read some of the properties have been defaulted e.g. `OrderStatusId`

Reply:
👍